CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.14.2] - 2025-02-12
+
```

Comment:
The changes should first just be added under [Unreleased], the new version header will be added in the version bump PR.

Reply:
Moved to Unreleased